Sources/SwiftDocCUtilities/Action/Actions/Convert/ConvertAction.swift

```diff
@@ -469,14 +469,22 @@ public struct ConvertAction: Action, RecreatingContext {
         }
 
         var didEncounterError = analysisProblems.containsErrors || conversionProblems.containsErrors
-        if try context.renderRootModules.isEmpty {
+        if (try context.renderRootModules + context.rootTechnologies).isEmpty {
+            // Check if the converted documentation is a tutorial to provide
+            // a specific error message.
+            let hasTutorial = context.knownPages.contains(where: {
+                guard let kind = try? context.entity(with: $0).kind else { return false }
+                return kind == .tutorial || kind == .tutorialArticle
+            })
             postConversionProblems.append(
                 Problem(
                     diagnostic: Diagnostic(
                         severity: .warning,
                         identifier: "org.swift.docc.MissingTechnologyRoot",
-                         summary: "No TechnologyRoot to organize article-only documentation.",
-                         explanation: "Article-only documentation needs a TechnologyRoot page (indicated by a `TechnologyRoot` directive within a `Metadata` directive) to define the root of the documentation hierarchy."
+                         summary: "There was no root found for this documentation catalog.",
+                         explanation: hasTutorial ? """
+                        For Tutorials please add a table of contents page (indicated by a `Tutorials` directive with the corresponding `Intro` and `Chapters` directive) to define the root of the documentation hierarchy. \n
```

Comment:
Out of curiosity, do you think it would be helpful to add a link to https://www.swift.org/documentation/docc/tutorials?

Reply:
> Is there a precedent with adding documentation links in our diagnostics? I am not sure that is something we want to do, it would mean we need to make sure the link remains up to date. @d-ronnqvist what do you think?

From https://github.com/apple/swift-docc/pull/664#discussion_r1275305566